### DISCUSSION TITLE
SFTP Provide Disk Used & Total Size

### DISCUSSION THREAD

**Discussion by HulkHogan6262** [pelican-dev/panel#255] [Ideas]
All is in the title

For example a software like [Mountain Duck](https://mountainduck.io/) allows mounting SFTP storage.

Because Wings does not provide anything, the software displays the stats of the local computer's cache disk and not the ones of the server on Pelican (e.g. The server has 5 GiB allocated and 900 MiB used, and the software displays a drive of 476 GiB (my local drive) and 398 GiB free space (399 GiB free on local drive)).

![image](https://github.com/pelican-dev/panel/assets/93836385/56c81f1c-e769-47fc-99d4-ff7a51446d94)
![image](https://github.com/pelican-dev/panel/assets/93836385/1666b74c-24c3-4f64-8ebd-d8027bb3852d)

**Comment by iamkubi** [comment]
This is a pretty niche use case which I don't really have any interest in supporting, nor the ability to recreate.  Your screenshot of a windows file explorer window that is sftp mounting your pterodactyl server as a disk isn't anything that can be easily reproduced.  If you can provide linux terminal commands that demonstrate the problem then I might look into it.
